src/model/spec/userProfile.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai';
+import * as deepFreeze from 'deep-freeze';
+import { UserProfile } from '../userProfile'
+import {} from 'mocha'
+import {} from 'core-js'
+
+describe('userProfile', () => {
+  it('Is instantiated and exists', () => {
+    // Arrange
+    // Act
+    const userProfile = new UserProfile();
```

Comment:
I use enzyme feature for React components (shallow and mount), is it worth using it for normal classes / objects? What would be the advantage? If in this case it's better I'm keen on using them for this cases as well.